package/templates/amzn2023.spec.template

```diff
@@ -0,0 +1,103 @@
+{# Amazon Linux 2023 Complete RPM Spec Template #}
+%bcond_without check
+
+Name:           mount-s3
+Version:        {{ version }}
+Release:        amzn2023
+Summary:        Mountpoint for Amazon S3
+
+License:        Apache-2.0
+URL:            https://github.com/awslabs/mountpoint-s3 
+Source0:        mountpoint-s3-{{ version }}.tar.gz
+Source1:        LICENSE
+Source2:        NOTICE
+Source3:        THIRD_PARTY_LICENSES
+
+{# Build dependencies #}
+BuildRequires:  clang
+BuildRequires:  clang-devel
+BuildRequires:  rust >= {{ rust_version }}      
+BuildRequires:  cargo >= {{ rust_version }}    
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  gcc-c++
+BuildRequires:  git
+BuildRequires:  pkgconfig
+BuildRequires:  fuse-devel
+BuildRequires:  glibc-devel
+BuildRequires:  glibc-headers
+BuildRequires:  glibc-static
+BuildRequires:  libstdc++-devel
+BuildRequires:  nasm
+BuildRequires:  make
+BuildRequires:  which
+BuildRequires:  rust-packaging
+BuildRequires:  rust-toolset
+
+{# Supported architectures #}
+ExclusiveArch: x86_64 aarch64
+
+{# Bundled C/C++ libraries for security tracking #}
+{% for lib_name, lib_version in submodule_versions.items() %}
+Provides: bundled({{ lib_name }}) = {{ lib_version }}
+{% endfor %}
+
+{# Runtime dependencies #}
+Requires:       ca-certificates
+Requires:       fuse >= 2.9.0
+Requires:       fuse-libs >= 2.9.0
+
+%description
+Mountpoint for Amazon S3 is a simple, high-throughput file client for
+mounting an Amazon S3 bucket as a local file system. With Mountpoint for Amazon
+S3, your applications can access objects stored in Amazon S3 through file
+operations like open and read. Mountpoint for Amazon S3 automatically
+translates these operations into S3 object API calls, giving your applications
+access to the elastic storage and throughput of Amazon S3 through a file
+interface.
+
+{# Build commands #}
+%build
+export CFLAGS="${CFLAGS:-%{optflags}} -O2 -Wno-error=cpp"
+export CMAKE_C_FLAGS="$CFLAGS"
+
+export MOUNTPOINT_S3_AWS_RELEASE="amzn2023"
```

Comment:
This looks wrong. The environment variable should be `MOUNTPOINT_S3_AWS_RELEASE_TARGET`.